tests/test_mera.py

```diff
@@ -0,0 +1,128 @@
+import pytest
+import numpy as np
+import pandas as pd
+from pathlib import Path
+
+import mera
+
+MASK = pd.read_csv(Path(__file__).parent / "resources" / "mask.csv", index_col=0)
+
+
+@pytest.fixture(scope="module")
+def obs_sim_df():
+    # Load the data
+    data_dir = Path(__file__).parent.parent / "mera/example/resources"
+
+    obs_df = pd.read_csv(data_dir / "im_obs.csv", index_col=0)
+    sim_df = pd.read_csv(data_dir / "im_sim.csv", index_col=0)
+
+    return obs_df, sim_df
+
+
+@pytest.fixture(scope="module")
+def ims(obs_sim_df: tuple[pd.DataFrame, pd.DataFrame]):
+    obs_df, sim_df = obs_sim_df
+
+    # Use every third period to speed up the test
+    ims = [
+        cur_im
+        for cur_im in np.intersect1d(obs_df.columns, sim_df.columns)
+        if cur_im.startswith("pSA")
+    ][::3]
+
+    return ims
+
+
+@pytest.fixture(scope="module")
+def res_df(obs_sim_df: tuple[pd.DataFrame, pd.DataFrame], ims: list[str]):
+    obs_df, sim_df = obs_sim_df
+
+    assert (
+        np.all(obs_df.index == sim_df.index)
+        and np.all(obs_df.event_id == sim_df.event_id)
+        and np.all(obs_df.stat_id == sim_df.stat_id)
+    )
```

Comment:
why not use pandas assert array equal here?
```suggestion
    pd.testing.assert_frame_equal(
        obs_df['event_id', 'stat_id'],
        sim_df['event_id', 'stat_id']
    )
```